src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import db.FruitStorage;
+import java.util.Map;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReportFromDB() {
+        StringBuilder stringBuilder = new StringBuilder()
+                .append("fruit,quantity");
+
+        for (Map.Entry<String, Integer> entry : FruitStorage.storage.entrySet()) {
+            stringBuilder.append(System.lineSeparator())
+                    .append(entry.getKey()).append(",").append(entry.getValue());
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)